frontend/src/Editor/Components/Text.jsx

```diff
@@ -4,10 +4,32 @@ import DOMPurify from 'dompurify';
 export const Text = function Text({ height, properties, styles, darkMode }) {
   const [loadingState, setLoadingState] = useState(false);
 
-  const { textSize, textColor, textAlign, visibility, disabledState } = styles;
+  let {
+    textSize,
+    textColor,
+    textAlign,
+    visibility,
+    disabledState,
+    fontWeight,
+    decoration,
+    transformation,
+    fontStyle,
+    lineHeight,
+    textIndent,
+    letterSpacing,
+    wordSpacing,
+    verticalShadow,
+    horizontalShadow,
+    shadowColor,
+    blur,
+    fontVariant,
+  } = styles;
 
+  verticalShadow = `${verticalShadow ?? '0'}px`;
+  horizontalShadow = `${horizontalShadow ?? '0'}px`;
+  blur = `${blur ?? '0'}px`;
   const text = properties.text === 0 || properties.text === false ? properties.text?.toString() : properties.text;
-
+  console.log('shadow', verticalShadow, horizontalShadow, blur, shadowColor);
```

Comment:
Please remove this console log, and while removing it there are some lint errors that are occurring, please fix those also.